src/main/java/baseball/view/Message.java

```diff
@@ -0,0 +1,25 @@
+package baseball.view;
+
+public final class Message {
+
+    public final static Integer RESTART = 1;
```

Comment:
final static, static final은 동일합니다.
하지만 일반적으로 권장되는 가이드라인이 있으니, 이에 맞춰 주시는 것이 좋습니다.

똑같이 sonarLint 툴에서 나온 경고문구를 그대로 옮깁니다.
```
The Java Language Specification recommends listing modifiers in the following order:
    //Annotations
    //public
    //protected
    //private
    //abstract
    //static
    //final
    //transient
    //volatile
    //synchronized
    //native
    //default
    //strictfp
    //Not following this convention has no technical impact, but will reduce the code’s readability because most developers are used to the standard order.
```